src/main/resources/application.yml

```diff
@@ -168,3 +168,6 @@ kafka:
       topic-pattern: (${environment}\.)(.*\.)inventory\.holdings-record
       concurrency: ${KAFKA_EVENTS_CONCURRENCY:2}
       data-type: org.folio.innreach.dto.Holding
+  backoff:
+    interval: 3600000
+    max_failure: 12
```

Comment:
These values are almost certainly not correct. Without these values being exactly right the entire solution that we have come up with won't work. Please read the documentation for these values and come up with something that will work for our purposes.

As I have stated elsewhere: we want it to retry for at least 12 hours and retry needs to not conflict with okhttp3.